plugins/link-show-direction.js

```diff
@@ -1,17 +1,24 @@
 // @author         jonatkins
 // @name           Direction of links on map
 // @category       Tweaks
-// @version        0.2.1
+// @version        0.2.2
 // @description    Show the direction of links on the map by adding short dashes to the line at the origin portal.
 
 
+/* exported setup --eslint */
+/* global L, dialog, addHook, map, links */
 // use own namespace for plugin
-window.plugin.linkShowDirection = function() {};
-window.plugin.linkShowDirection.ANIMATE_UPDATE_TIME = 1000; // 1000ms = 1s
+var linkShowDirection = {};
+window.plugin.linkShowDirection = linkShowDirection;
+
+// exposed
+// linkShowDirection.showDialog = showDialog;
+
+var ANIMATE_UPDATE_TIME = 1000; // 1000ms = 1s
 
 // Hack:
 // 100000 - a large enough number to be the equivalent of 100%, which is not supported Leaflet when displaying with canvas
-window.plugin.linkShowDirection.styles = {
+var styles = {
```

Comment:
Someone may want to override the styles, so better expose them.

Reply:
exposed